src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py

```diff
@@ -260,13 +259,7 @@ def loop_body(step, args):
 
         # scale the initial noise by the standard deviation required by the scheduler
         latents = latents * self.scheduler.init_noise_sigma
-
-        if debug:
-            # run with python for loop
-            for i in range(num_inference_steps):
-                latents, scheduler_state = loop_body(i, (latents, scheduler_state))
```

Comment:
Think it's nice to have this option for easy debugging.

Reply:
The problem is that `debug` has to be passed from everywhere and it adds noise to the very busy `partial` here: https://github.com/huggingface/diffusers/blob/7db319bdae4023bbd6c7a7ca3d16abd873dcdbd0/src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py#LL388-L391

How do you feel about using a `DEBUG` constant instead, that we only use when we are really debugging?

Reply:
I see, yeah setting DEBUG as a constant sounds good too!